packages/merge/src/modules/merge/sagas.js

```diff
@@ -76,13 +82,13 @@ export function* getMergeBody() {
 }
 
 export function* executeMerge() {
-  const resource = 'nice2/rest/merge/merge'
+  const resource = 'merge/merge'
   const options = {
     method: 'POST',
     body: yield call(getMergeBody)
   }
 
   const response = yield call(rest.requestSaga, resource, options)
-
-  yield put(actions.setMergeResponse(response))
+  yield put(actions.setMergeResponse(response.body.notCopiedRelations, response.body.notDeletedEntities,
```

Comment:
with the change from above this would look like this: 

`yield put(actions.setMergeResponse(response.body))`